script/test/cmd/tests.sh

```diff
@@ -173,4 +173,39 @@ convert::check_artifacts_generated "kompose -f $KOMPOSE_ROOT/examples/docker-com
 # Behavior with -o <dirname>/<filename>
 convert::check_artifacts_generated "kompose -f $KOMPOSE_ROOT/examples/docker-compose.yml convert -o $TEMP_DIR/output_file -j" "$TEMP_DIR/output_file"
 
+#####
+# Test the preference file behavior
+
+# valid preference file1
+# kompose convert; reads from pref-file
```

Comment:
what preference file? i see `--profile default` passed but no actual profile it links to.